### ISSUE TITLE
indicate if Event is Stashed on API

### ISSUE THREAD

**Issue by warmfusion** [sensu/sensu#999]
Scenario:
I want to be able to ping the Events API and get a list of the currently active Events but also have an indication if the events stashed.

Can the Events API be extended to include this information as shown where a `stashed` boolean is included alongside `handle` flag.

Perhaps the `stash` hash could be included which includes the same kind of information you'd get from the Stashes API; Though if the path is given this could be up to the user to extract the stash information if they want it. Effectively; I want to be told that a given Event has been Stashed by something (and therefore don't need to worry about it), and if I can be told what that something(s) might be all the better.

```
{
  "check": {
    "aggregate": true,
    "command": "/bin/false'",
    "duration": 0.037,
    "executed": 1436361555,
    "handle": false,
    "interval": 60,
    "issued": 1436361555,
    "name": "never-fails",
    "output": "Oh Noes - Its dead",
    "standalone": true,
    "stashed": true,
# OPTIONAL
    "stashes": [
      {
        "content": {
          "reason": "Keeps Breaking",
          "source": "uchiwa",
          "timestamp": 1436572563
        },
        "expire": -1,
        "path": "silence/my-broken-server.example.com/never-fails"
      }
    ],
# Alternative
    "stashes": [
      "silence/my-broken-server.example.com/never-fails"
    ],
# END OPTIONAL
    "status": 2,
    "tip": "This is a custom field you know"
  },
  "client": "my-broken-server.example.com"
}
```

**Comment by calebhailey** [comment]
@warmfusion hi there! Thanks for submitting this issue! I've tagged this for renewed discussion around a subdue redux.

**Comment by cwjohnston** [comment]
I believe this is closed by #1400 which adds `silenced_by` attribute to events. This attribute provides a list of silence entries by ID, the content of which can be looked up via the `/silenced` API.

**Comment by warmfusion** [comment]
Agreed.

Closing 👍
